docs/examples/advanced/asymmetry_comparison_advanced.py

```diff
@@ -0,0 +1,122 @@
+"""
+Complex asymmetry
+=================
+
+This example shows how to plot the asymmetry between 2 histograms.
+"""
+
+
+# Get dummy data
+from plothist.generate_dummy_data import generate_dummy_data
+
+df = generate_dummy_data()
+
+###
+from scipy.stats import norm
+import numpy as np
+from plothist import (
+    create_comparison_figure,
+    make_hist,
+    plot_error_hist,
+    plot_function,
+    plot_comparison,
+    add_luminosity,
+    add_text,
+)
+
+
+# Define some random functions that will be used as fit functions
+def f1(x):
+    return 4000 * norm.pdf(x, loc=-0.5, scale=1.6)
+
+
+def f2(x):
+    return 4000 * norm.pdf(x, loc=0.5, scale=1.6)
+
+
+name = "variable_1"
+category = "category"
+
+x1 = df[name][df[category] == 5]
+
+x_range = [-9, 9]
+
+# Create the histograms used as data
+h1 = make_hist(x1 - 2.5, bins=50, range=x_range)
+h2 = make_hist(x1 - 1.5, bins=50, range=x_range)
+
+# Create the figure
+fig, (ax_main, ax_comparison) = create_comparison_figure(
+    gridspec_kw={"height_ratios": [2, 1]}
+)
+
+# Define the marker style
+marker_1 = {
+    "color": "tab:red",
+    "markeredgecolor": "tab:red",
+    "ls": "None",
+    "fmt": "o",
+    "markersize": 5,
+    "label": r"$Event^{+}$",
+}
+
+marker_2 = {
+    "color": "tab:blue",
+    "markerfacecolor": "white",
+    "markeredgecolor": "tab:blue",
+    "ls": "None",
+    "fmt": "o",
+    "markersize": 5,
+    "label": r"$Event^{-}$",
+}
+
+# Plot the data
+plot_error_hist(
+    h1,
+    ax_main,
+    **marker_1,
+)
+
+plot_error_hist(
+    h2,
+    ax_main,
+    **marker_2,
+)
+
+# Plot the functions
+plot_function(f1, x_range, ax_main, color=marker_1["color"], label="$Fit^+$")
+plot_function(
+    f2, x_range, ax_main, color=marker_2["color"], linestyle="--", label="$Fit^-$"
+)
+
+# Plot the asymmetry comparison between the 2 histograms
+plot_comparison(
+    h1,
+    h2,
+    ax_comparison,
+    comparison="asymmetry",
+    comparison_ylim=(-1, 1),
+    comparison_ylabel="Asymmetry",
+)
+
+# Define the asymmetry of the 2 functions
+asymmetry = lambda x: (f1(x) - f2(x)) / (f1(x) + f2(x))
+
+# Plot the asymmetry of the 2 functions
+plot_function(asymmetry, x_range, ax_comparison, color="black")
+
+ax_main.legend()
+
+fig.align_ylabels()
+
+ax_main.set_xlim(x_range)
+ax_main.set_ylim(ymin=0)
+ax_main.set_ylabel("Candidates")
+ax_main.legend()
+
+ax_comparison.set_xlim(x_range)
+ax_comparison.set_xlabel(name)
+
+add_luminosity(collaboration="LMN 3", ax=ax_main, is_data=False, preliminary=True)
+
+fig.savefig("asymmetry_comparison_advanced.svg", bbox_inches="tight")
```

Comment:
Maybe not for this PR, but I would by in favor of un-capitalizing the s of simulation and un-bolding simulation.